### ISSUE TITLE
Snake doesn't move smootly

### ISSUE THREAD

**Issue by RowanHarley** [iiegor/slither#22]
Whenever I go to play a game my snake doesn't move very well and no food comes up. Don't know if this is just me or something. Here's a screenshot:
![screenshot 1](https://cloud.githubusercontent.com/assets/16364197/14716605/95a20ad4-07e5-11e6-8420-74dfeafaa15f.png)
The snake just continues to move like this and I get this in the console:
![screenshot 2](https://cloud.githubusercontent.com/assets/16364197/14716630/b954fdba-07e5-11e6-8a53-56c927821666.png)

**Comment by iiegor** [comment]
I think this is due to a bad calculation of the snake direction or position. Although I'm aware of this, I'll keep this open for anyone who wants to share anything related with this or a possible solution.

**Comment by RowanHarley** [comment]
Is this just a bug with my computer or is this a bug with the code

**Comment by techpulse** [comment]
@iiegor i can't even run it, my snake is completely stuck

**Comment by RowanHarley** [comment]
As much as I'd love to help try fix this bug, I know nothing about coffeescript or game re-programming

**Comment by Qasimmodo** [comment]
It's a bug within the code, the project isn't finished yet...

**Comment by morenja** [comment]
me too have that problem.

**Comment by iiegor** [comment]
The commit 323d95e2290e6420a1d158999636c9b8607384a0 fixes this, although, I will keep this open because the way was fixed is not the correct.

**Comment by RowanHarley** [comment]
Ok @iiegor just tested it and that bug is almost gone! The only time it doesn't move smoothly is when I try go around in circles. Also playing it, I don't see any food to eat

**Comment by RowanHarley** [comment]
The snake can't speed up as well, even though it says it in the console

**Comment by iiegor** [comment]
@RowanHarley yes, in the latest release I temporarily removed food spawning due to big performance issues and bad spawn position of them. Features like the speed boost can't be developed right now since the basic movement of the snake isn't done yet.

**Comment by RowanHarley** [comment]
ok just thought that was a bug that came up because of a file changed

**Comment by Qasimmodo** [comment]
I believe it has been resolved now, but as @iiegor said the speed-boost and pellets will not be implemented yet, because they're not at that stage.

**Comment by RowanHarley** [comment]
Dont think I'll close this though as its still a bit buggy when trying to do circles

**Comment by vancourt** [comment]
how to connect using slitherio website?

**Comment by RowanHarley** [comment]
@vancourt hold Ctrl+Shift+J then type sos = [{ ip: "localhost", po: 443, ac: 34, ptm: 121 }]

**Comment by RowanHarley** [comment]
And by the way this really isn't the place to ask that. If you have a problem you should create an issue

**Comment by InvincibleMiners** [comment]
http://bit.ly/SlitherSnakeFix
